frontend/src/pages/ServicePage/ServicePage.tsx

```diff
@@ -111,7 +111,7 @@ export const ServicePage: React.FC = () => {
       </Columns>
       <Divider />
       <List>
-        <UsersSelect service={service} />
+        <UsersSelect device={device} service={service} />
```

Comment:
Here we will have to just do either device OR service but not both so we can pull specific info.
```suggestion
        <UsersSelect service={service} />
```